api/modules/users/validators.py

```diff
@@ -13,7 +15,7 @@ def _has_at(email):
     :param email:
     :return: boolean
     """
-    return "@" in email
+    return '@' in parseaddr(email)[1]
```

Comment:
Do not modify existing validators, instead add a new one.
Also add tests for the same